samples/nrf9160/modem_shell/prj.conf

```diff
@@ -73,6 +73,8 @@ CONFIG_NET_SOCKETS_POLL_MAX=10
 CONFIG_NRF_MODEM_LIB=y
 CONFIG_NRF_MODEM_LIB_SHMEM_TX_SIZE=16384
 CONFIG_NRF_MODEM_LIB_SHMEM_RX_SIZE=8192
+# Modem library fault handling
+CONFIG_NRF_MODEM_LIB_ON_FAULT_APPLICATION_SPECIFIC=y
```

Comment:
Because the modem_shell handler only prints the crash reason, I think we could remove this and use the default implementation in modem lib.